docs/examples/schema1.py

```diff
@@ -31,11 +30,4 @@ class MainModel(BaseModel):
     class Config:
         title = 'Main'
 
-debug(MainModel.schema())
```

Comment:
Now we can't run this script as is because `debug` is not defined function. I think as example we can show only usage of `.schema_json()`, but also I think can extend documentation about methods which we can use to generate schemas.

Reply:
I guess just use print not debug.

Or we could advertise python-devtools :wink:.

Reply:
Let's keep it simple :). Renamed `debug` to `print`.